plugin.video.longnow/resources/language/English/strings.po

```diff
@@ -0,0 +1,60 @@
+msgid ""
```

Comment:
language folder layout is wrong: resources/language/resource.language.en_gb/strings.po